tests/conditional_test.py

```diff
@@ -28,7 +30,14 @@
 from pytket.qir.conversion.api import QIRFormat, pytket_to_qir
 
 
-def test_pytket_qir_conditional() -> None:
+@pytest.mark.parametrize(
```

Comment:
This file now contains the tests from `tests/conditional_test.py` and `tests/conditional_test.py` with parametrised tests. I Most of the tests have been in both files anyway, I have added the tests that were in only on of the files. I had to rename a few of the tests to make sure I can have all of the in one file.